libs/api/src/main/java/de/cyface/api/MeasurementIterator.java

```diff
@@ -227,7 +219,7 @@ public Future<Measurement> next() {
      */
     private void resolveNext(final List<TrackBucket> buckets, final MeasurementIdentifier nextMeasurementId,
             final TrackBucket nextBucket) {
-        final var measurement = measurement(buckets);
+        final var measurement = new Measurement(buckets);
```

Comment:
This constructor code moved to serialization/libs/model/Measurement, as you can see (also by the deleted lines below)